contracts/OlasMech.sol

```diff
@@ -71,9 +75,14 @@ abstract contract OlasMech is Mech, IErrorsMech, ImmutableStorage {
         address _mechMarketplace,
         address _serviceRegistry,
         uint256 _serviceId,
-        uint256 _maxDeliveryRate,
+        uint256 _maxDeliveryRate, // TODO - for subscriptions, why do we even specify this here; it should be specified by the requester
```

Comment:
To discuss

Reply:
It's still the mech level - so mech must decide its max delivery rate.

Reply:
we'll have a call about this